install/0000_30_machine-api-operator_13_baremetalhost_configmap.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ironic-bmo-configmap
+data:
+  http_port: "6180"
+  provisioning_interface: "ens3"
+  provisioning_ip: "172.22.0.3/24"
+  dhcp_range: "172.22.0.10,172.22.0.100"
+  deploy_kernel_url: "http://172.22.0.3:6180/images/ironic-python-agent.kernel"
+  deploy_ramdisk_url: "http://172.22.0.3:6180/images/ironic-python-agent.initramfs"
+  ironic_endpoint: "http://172.22.0.3:6385/v1/"
+  ironic_inspector_endpoint: "http://172.22.0.3:5050/v1/"
+  cache_url: "http://172.22.0.1/images"
+  rhcos_image_url: "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190520.0/"
```

Comment:
This image shouldn't be hard-coded, it needs to be derived from the release payload?

Reply:
Thanks for pointing that out. I am working on figuring out how to do this.